### ISSUE TITLE
Cannot unmarshal `extension/forwarding-number` result

### ISSUE THREAD

**Issue by grokify** [grokify/go-ringcentral-client#20]
# Errors

Found the following errors in https://github.com/grokify/ringcentral-legacy-api-proxy

1. `json: cannot unmarshal array into Go struct field ForwardingNumberInfo.features of type string`
1. `json: cannot unmarshal string into Go struct field ForwardingNumberInfo.flipNumber of type int32`

## ForwardingNumberInfo Definition Error 1

The following error occurs because `ForwardingNumberInfo.features` should be `[]string`, not `string`.

`json: cannot unmarshal array into Go struct field ForwardingNumberInfo.features of type string`

**Current Definition**

```yaml
    ForwardingNumberInfo:
      type: "object"
      properties:
        id:
          type: "string"
          description: "Internal identifier of a forwarding/call flip phone number"
        uri:
          type: "string"
          description: "Canonical URI of a forwarding/call flip phone number"
        phoneNumber:
          type: "string"
          description: "Forwarding/Call flip phone number"
        label:
          type: "string"
          description: "Forwarding/Call flip number title"
        features:
          type: "string"
          description: "Type of option this phone number is used for. Multiple values are accepted"
          enum:
            - "CallFlip"
            - "CallForwarding"
        flipNumber:
          type: "integer"
          description: "Number assigned to the call flip phone number, corresponds to the shortcut dial number"
```

**Possible Corrected Definition**

Looks like the following should work:

https://stackoverflow.com/questions/36888626/defining-enum-for-array-in-swagger-2-0

```yaml
AnArray:
  type: array
  items:
    type: string
    enum:
      - MALE
      - FEMALE
      - WORKER
```

For:

```yaml
        features:
          type: "array"
          description: "Type of option this phone number is used for. Multiple values are accepted"
          items:
            type: string
            enum:
              - "CallFlip"
              - "CallForwarding"
```

**Comment by grokify** [comment]
## ForwardingNumberInfo Definition Error 2

`json: cannot unmarshal string into Go struct field ForwardingNumberInfo.flipNumber of type int32`

**Comment by grokify** [comment]
Fixed in f12f4c59408ca1def30ca6cdf10c3841b1261b88

Spec changes: https://github.com/grokify/go-ringcentral/commit/f12f4c59408ca1def30ca6cdf10c3841b1261b88#diff-3253245913faccbdf3cccacd218fe678
